packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -60,9 +56,20 @@ const addWebRender = (content, authProvider) => {
 }
 
 // the files to create to support auth
-export const files = () => {
+export const files = (provider) => {
+  let file = TEMPLATE
+  try {
+    let providerTemplatePath = path.resolve(
+      __dirname,
+      'templates',
+      `auth.js.template_${provider}`
+    )
+    file = fs.readFileSync(providerTemplatePath).toString()
+  } catch {
```

Comment:
I'm not a huge fan adding error handling for something that will happen more often that not (3 of the 4 generators won't have a template and will actually raise an error here). Could this be better handled with a `fs.existsSync()` check and only read in the template if it exists?

Reply:
I fetched the templates based on what's available and pushed the appropriate commit, please tell me if it's ok or if I should fix it.